pycbc/conversions.py

```diff
@@ -1007,7 +1220,8 @@ def nltides_gw_phase_diff_isco(f_low, f0, amplitude, n, m1, m2):
            'spin2y_from_mass1_mass2_xi2_phi_a_phi_s',
            'chirp_distance', 'det_tc', 'snr_from_loglr',
            'freq_from_final_mass_spin', 'tau_from_final_mass_spin',
+           'final_spin_from_f0_tau', 'final_mass_from_f0_tau',
            'optimal_dec_from_detector', 'optimal_ra_from_detector',
            'chi_eff_from_spherical', 'chi_p_from_spherical',
-           'nltides_gw_phase_diff_isco'
+           'nltides_gw_phase_diff_isco',
```

Comment:
Why did you add here a comma before the end of the list? Is this needed?

Reply:
This is so that you don't get a merge conflict every time someone adds a new function to the list.